content/docs/0.8.1/snapshots-and-backups/backup-and-restore/backupstores-and-backuptargets.md

```diff
@@ -108,6 +108,13 @@ Please follow [the Kubernetes document](https://kubernetes.io/docs/concepts/conf
 
 Notice the secret must be created in the `longhorn-system` namespace for Longhorn to access.
 
+### Using a self signed SSL certificate for S3 communication
+If you want to use a self signed ssl certificate you can expose an AWS_CERT environment variable that contains your public certificate to the longhorn pods.
+we recommend you do so by adding the AWS_CERT key/val to the `minio-secret` or `aws-secret` and making it available to the `longhorn-system` namespace.
+For an example of using a self signed ssl certificate, have a look at our `minio-backupstore.yaml` linked above.
```

Comment:
YAML file wasn't updated.

Reply:
I added the AWS_CERT under the minio secret. If I remember right you mentioned the yaml will get updated from the longhorn-test repro, where I committed the updated yaml?

Reply:
I meant, the document should also show the code snippet for user, just like the minio-backuostore.yaml a few lines above. You can add cert section above, or show it here. Either is fine.